javascript-light-bulb.tar/javascript-light-bulb/styles.css

```diff
@@ -0,0 +1,15 @@
+#container{
+   background-color: black;
+  width:1000px;
+  height:500px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
you could have a couple classes here to toggle between in the on/off state.

ex:

```css
.container {
   /* this could be the default color in the "off" state" */
   background-color: black; 
}
.is-on {
   /* add the is-on class when you turn on the light bulb, and remove it when it's "off" */
   background-color: rgb(252, 234, 134);
}
```